### PR TITLE
Use a unique name for the version properties file.

### PR DESCRIPTION
**Pull request by paulblessing**
Other libraries may include a `version.properties` file which could cause a lookup failure for `robolectric.version`.

### CODE REVIEW

File: robolectric/src/main/java/org/robolectric/internal/SdkConfig.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ private DependencyJar createDependency(String groupId, String artifactId, String
   private static String getRobolectricVersion() {
     try {
       final Properties properties = new Properties();
-      properties.load(SdkVersion.class.getClassLoader().getResourceAsStream("version.properties"));
+      properties.load(SdkVersion.class.getClassLoader().getResourceAsStream("robolectric.version.properties"));
```
Comment (erd) [inline robolectric/src/main/java/org/robolectric/internal/SdkConfig.java]: How about `robolectric-version.properties`?

Comment (paulblessing) [inline robolectric/src/main/java/org/robolectric/internal/SdkConfig.java]: Makes no difference to me. Is the convention for the project to amend the commit with a force push, or to add a second commit?

Comment (erd) [inline robolectric/src/main/java/org/robolectric/internal/SdkConfig.java]: I'd prefer a force push so it's a single commit.

**Comment by paulblessing** [comment]
Rebased down to a single commit.

**Pull request merged**